### PR TITLE
Update roctx.cpp

### PR DESCRIPTION
**Pull request by rkebichi**
Changed returned value from roctxRangePush and Pop to stack size

### CODE REVIEW

File: src/roctx/roctx.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ PUBLIC_API int roctxRangePushA(const char* message) {
   if (api_callback_fun) api_callback_fun(ACTIVITY_DOMAIN_ROCTX, ROCTX_API_ID_roctxRangePushA, &api_data, api_callback_arg);
   message_stack.push(strdup(message));
   API_METHOD_CATCH(-1);
-  return 0;
+  return message_stack.size();
```
Comment (eshcherb) [inline src/roctx/roctx.cpp]: Should be 'message_stack.size() - 1', as according to documentation it is zero based and we want the correspondent push/pop pairs to return the same value.

Comment (rkebichi) [reply src/roctx/roctx.cpp]: indeed. I will correct that.

File: src/roctx/roctx.cpp [hunk 2]
```diff
@@ -141,7 +141,7 @@ PUBLIC_API int roctxRangePop() {
       message_stack.pop();
   }
   API_METHOD_CATCH(-1)
-  return message_stack.size();
+  return message_stack.size()-1;
```
Comment (eshcherb) [inline src/roctx/roctx.cpp]: I think it was correct, no need to subtract 1 here as it is after popping.

Comment (rkebichi) [reply src/roctx/roctx.cpp]: I checked the doc and you are right. Will fix it.

**Pull request merged**